scripts/integration/docker-compose.prometheus.yml

```diff
@@ -0,0 +1,60 @@
+services:
+  influxdb:
```

Comment:
🤔 why do we start influxdb for prometheus tests?

Reply:
It's required for the `remote_write` sink [here](https://github.com/vectordotdev/vector/blob/master/src/sinks/prometheus/remote_write.rs#L470)

Reply:
Seems weird/unnecessary 🤷

Reply:
The prometheus server doesn't actually implement the remote_write protocol, while influxdb does, so I used it to validate the protocol compliance against something other than our own source.

Reply:
it does as of today, `docker run --rm -it prom/prometheus --enable-feature=remote-write-receiver --config.file=/etc/prometheus.yml^C`

Reply:
Ah, right, I'm forgetting my history again. When I wrote the remote-write setup, I had not found that, and the prometheus documentation seemed (to me at least) to point away from itself. So at this point it is a historical artifact that should probably be rectified.

Reply:
I mean, to be fair, it seems pretty undocumented and unclear how stable/supported that is today 🤷

Reply:
@spencergilbert or @bruceg, do you mind if we open an issue to tackle that in a latter PR and we keep this one as it is?

Reply:
Absolutely that belongs in a separate issue. It's completely unrelated to this change.